cdk/src/open-data-platform/data-plane/data-import/write-water-systems-data.handler.ts

```diff
@@ -28,13 +28,15 @@ async function insertBatch(
     secretArn: process.env.CREDENTIALS_SECRET ?? '',
     sql: `INSERT INTO water_systems (pws_id,
                                      pws_name,
-                                     lead_connections_count,
+                                     lead_connections_low_estimate,
```

Comment:
I actually don't think we are getting this anymore. Just lead lines and service lines

Reply:
Hmm. Should we wait to create schema changes until we have the data and know what it looks like?

Reply:
Is this documented Justine? I can hold off submitting until we get sources but it will block e2e testing if we are using the "new" schema in API.

Reply:
Updated but still awaiting Jared's response.